agentwire/__main__.py

```
@@ -6841,6 +6842,88 @@ def cmd_projects_list(args) -> int:
+        return _fail("Invalid project name (allowed: letters, digits, '.', '_', '-')")
+
+    projects_dir = get_config().projects.dir.expanduser().resolve()
+    projects_dir.mkdir(parents=True, exist_ok=True)
+    project_path = projects_dir / name
+
+    if project_path.exists():
+        return _fail(f"Project already exists at {project_path}")
+
+    try:
+        if clone_url:
+            result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)